src/main/kotlin/ar/edu/unsam/pds/models/User.kt

```diff
@@ -1,20 +1,30 @@
 package ar.edu.unsam.pds.models
 
-import ar.edu.unsam.pds.repository.Element
 import ar.edu.unsam.pds.exceptions.ValidationException
+import jakarta.persistence.*
 import java.util.UUID
 
+@Entity
 class User(
+    @Column(length = 30)
     var name: String,
+    @Column
     var lastName: String,
+    @Column
     var email: String,
+    @Column
     var image: String,
-) : Element {
-    val id: String = UUID.randomUUID().toString()
+)  {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    lateinit var id : UUID
 
+    @OneToMany(fetch= FetchType.EAGER, cascade = [CascadeType.ALL])
```

Comment:
nacho acá seria many to many me parece
un user se puede inscribir en muchos assignments
un assignment tiene muchos users inscriptos

Reply:
Perdon que no está el DER ayer no pude, si puedo a la noche lo hago. Estaria bueno tenerlo para usarlo de guia

Reply:
dale, espero a tener el der, tranqui. no hay apuro!